packages/garbo/src/fights.ts

```diff
@@ -2190,16 +2190,20 @@ export function estimatedFreeFights(): number {
   );
 }
 
-// Possible additional free fights from tentacles
-export function estimatedTentacles(): number {
+// Possible additional free fights from Eldritch Attunement
+export function estimatedAttunementTentacles(): number {
   const totalFreeFights =
     sum(freeFightSources, (source: FreeFight) => {
       const avail = source.tentacle ? source.available() : 0;
       return typeof avail === "number" ? avail : toInt(avail);
     }) +
     possibleFreeFightQuestTentacleFights() +
     possibleFreeGiantSandwormQuestTentacleFights();
-  return clamp(totalFreeFights, 0, 11); // Capped at 11 per day
+  return clamp(
+    totalFreeFights,
+    0,
+    11 - (get("questL02Larva") !== "unstarted" ? 1 : 0), // Capped at 11, minus one if we have access to Science Tent
```

Comment:
we should account for the skill as well. but also account for whether these have already been used